addons/website_sale/views/templates.xml

```diff
@@ -1355,6 +1355,26 @@
                                                 Add to cart
                                             </a>
                                         </div>
+                                        <div
+                                            id="contact_us_wrapper"
+                                            t-attf-class="{{'d-flex' if combination_info['prevent_zero_price_sale']
+                                                else 'd-none'}} oe_structure oe_structure_solo #{_div_classes}"
```

Comment:
spliting class attribute in multiple line does not increase readability you can keep it in one line as before